azurelinuxagent/common/exception.py

```diff
@@ -153,6 +153,14 @@ class ProtocolNotFoundError(ProtocolError):
     def __init__(self, msg=None, inner=None): # pylint: disable=W0235
         super(ProtocolNotFoundError, self).__init__(msg, inner)
 
+class IncompleteGoalStateError(ProtocolError):
+    """
+    Goal state is returned incomplete.
+    """
+
+    def __init__(self, msg=None, inner=None): # pylint: disable=W0235
```

Comment:
we need to document the reason of all new pylint supressions

Reply:
Good catch. Will do.

Reply:
thanks, also add 2 paces before the comment. pycharm is noisy about single space